pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -156,12 +155,11 @@ func (r *PerformanceProfileReconciler) removeOLMOperator() error {
 		subscription := &subscriptions.Items[i]
 		if subscription.Name == "performance-addon-operator" {
```

Comment:
IIRC we should not assume the name of the subscription to be `performance-addon-operator`

Reply:
@camilamacedo86  perhaps you can advise here.
In case we want to remove a 4.10 optional operator, following the [CLI steps](https://docs.openshift.com/container-platform/4.10/operators/admin/olm-deleting-operators-from-cluster.html) translated into code as suggested in this PR, what would be the cretin field of the subscription we can track and delete for our operator - `performance-addon-operator`.
In case  `Subscription.Name` field can be something other than `performance-addon-operator` , is there a field that will always contain the constant for `performance-addon-operator` for all types of deployments ? (CLI/UI/Manual)

Reply:
Probably you can find if the `subscription.Status.CurrentCSV` includes `performance-addon-operator`.

Reply:
I have changed the removal mechanism to get the csv first by the url